inst/include/cpp11/doubles.hpp

```diff
@@ -148,8 +148,7 @@ inline doubles as_double(sexp x) {
     return ret;
   }
 
-  // else
-  stop("Expected a numeric vector for");
+  throw type_error(INTSXP, TYPEOF(x));
```

Comment:
cpp11 defines a `type_error` class to generate a more informative error message for type issues, so we can use that here.

https://github.com/r-lib/cpp11/blob/23ae34fd6be84a3602e154fc3bf66897e117a38b/inst/include/cpp11/r_vector.hpp#L26-L39

Reply:
Okay awesome thank you!